### PR TITLE
[xcode11] Bump for beta 6

### PR DESCRIPTION
**Pull request by spouliot**
AVFoundation crash added to https://github.com/xamarin/xamarin-macios/issues/6212

### CODE REVIEW

File: src/avfoundation.cs [hunk 1]
```diff
@@ -3243,6 +3243,7 @@ interface AVCaptureSynchronizedMetadataObjectData
 
 	[NoMac, NoWatch, NoTV, iOS (11,0)]
 	[BaseType (typeof(AVCaptureSynchronizedData))]
+	[DisableDefaultCtor]
```
Comment (VincentDondain) [inline src/avfoundation.cs]: That's a breaking change

Comment (VincentDondain) [reply src/avfoundation.cs]:
```
Type Changed: AVFoundation.AVCaptureSynchronizedDepthData

Removed constructor:

	public AVCaptureSynchronizedDepthData ();
```

Comment (spouliot) [reply src/avfoundation.cs]: yes, it's logged in the xcode11 todo issue

Comment (VincentDondain) [reply src/avfoundation.cs]: Ah yes I see, cool

**Pull request merged**